src/renderer/components/+user-management-roles/add-role-dialog.tsx

```diff
@@ -64,13 +68,20 @@ export class AddRoleDialog extends React.Component<Props> {
             nextLabel="Create"
             next={this.createRole}
           >
+            <SubTitle title="Role Name" />
             <Input
               required autoFocus
-              placeholder={`Role name`}
```

Comment:
I prefer the old name as it is descriptive instead of instructive.

Reply:
I was simply following the convention that is used elsewhere when there's a subtitle and an input (particularly, creating a service account).

Reply:
Looking at the rest of the repo, I would say that slight majority is actually descriptive instead.

Reply:
Changed it to descriptive, but not fully the same name as the `Subtitle`. This is similar to what is shown when creating secrets.